src/domain/event/eventForm/OrderableEventCheckboxField.tsx

```diff
@@ -0,0 +1,27 @@
+import { FieldProps } from 'formik';
+import { CheckboxProps } from 'hds-react';
+import * as React from 'react';
+
+import CheckboxField from '../../../common/components/form/fields/CheckboxField';
+
+// This is its own component because how Formik recommends to handle situations
+// where form value is based on other form value.
+// see: https://github.com/formium/formik/issues/2204#issuecomment-574207100
```

Comment:
Is this field based on other form values? I can tell it has side effects which affect other form values.

Reply:
Hmm I think it is not based on other form values but rather automatically changed to different value based on other values

Reply:
But that is actually handled VirtualEventCheckboxField. Should be maybe moved into this component